From 61286b73e21a75492abc8672e3812da9f85bfc35 Mon Sep 17 00:00:00 2001
From: Eric Garver <e@erig.me>
Date: Mon, 5 Feb 2018 10:25:21 -0500
Subject: [PATCH] doc/xml/Makefile: firewallctl docs depend on errorcodes

Fixes the below build error:

$ make -C doc/xml ../html/firewallctl.html
...
xsltproc -o ../html/firewallctl.html --nonet --xinclude  transform-html.xsl firewallctl.xml
warning: failed to load external entity "errorcodes.xml"
firewallctl.xml:1937: parser error : Failure to process entity errorcodes
            &errorcodes;
                        ^
firewallctl.xml:1937: parser error : Entity 'errorcodes' not defined
            &errorcodes;
                        ^
unable to parse firewallctl.xml
---
 doc/xml/Makefile.am | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/doc/xml/Makefile.am b/doc/xml/Makefile.am
index be89d803..129ec501 100644
--- a/doc/xml/Makefile.am
+++ b/doc/xml/Makefile.am
@@ -46,9 +46,11 @@ all: $(man_MANS) $(HTMLS)
 clean:
 	-test -z "$(CLEAN_FILES)" || rm -f $(CLEAN_FILES)
 
-../man/man1/firewall-cmd.1: errorcodes.xml firewall-cmd.xml
+../man/man1/firewallctl.1: errorcodes.xml
+../html/firewallctl.html: errorcodes.xml
 
-../html/firewall-cmd.html: errorcodes.xml firewall-cmd.xml
+../man/man1/firewall-cmd.1: errorcodes.xml
+../html/firewall-cmd.html: errorcodes.xml
 
 ../man/man1/%.1: %.xml authors.xml notes.xml seealso.xml transform-man.xsl
 	$(XSLTPROC) -o $@ $(XSLTPROC_MAN_FLAGS) $<
